health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/UploadDataMapping.js

```
@@ -0,0 +1,677 @@
+              return item;
+            })
+          : state?.data?.map((item) => {
+              const BoundaryLoc = action.t(action?.schemas?.find((i) => i.description === "Boundary Code")?.name);
+              if (item?.[action.t("HCM_ADMIN_CONSOLE_FACILITY_CODE")] === action?.payload?.row?.[action.t("HCM_ADMIN_CONSOLE_FACILITY_CODE")]) {
+                return {
+                  ...item,
+                  [BoundaryLoc]: action?.payload?.selectedBoundary?.map((i) => i?.code)?.join(","),
+                };
+              }
+              return item;
+            });
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)